jobs/jobscheduler.py

```diff
@@ -349,7 +367,7 @@ def initJobs(jobs, resources, otype):
                 if (opts.maxmemory > 0)
                 else int(MachineMemoryGB * 1024 * 1024 * 10.24 * opts.memory)
             ),
-            "gpu": list(range(8)),
+            "gpu": CUDAGPUList + ROCMGPUList,
```

Comment:
@iarspider , I think IB relval job will hang and do nothing it both `CUDAGPUList`  and `ROCMGPUList` are empty  ( may be due to gpu/driver issue the `xxxComputeCapabilities` does not return any thing)